### PR TITLE
Update microsoft-teams-rooms-premium.md

### PR DESCRIPTION
**Pull request by altsou**
The managed service now supports new regions. 

Before merging, two things to be corrected: 
- Move Australia to line 92 (second in list)
- Insert Finland in line 107

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/rooms/microsoft-teams-rooms-premium.md [hunk 1]
```diff
@@ -86,29 +86,86 @@ For more details on the Microsoft Teams Rooms managed service support coverage,
 
 ## Supported regions
 
-The Microsoft Teams Rooms managed service is available in the following countries and regions.
+The Microsoft Teams Rooms managed service is available in the following countries and regions. 
 
-- Australia
+- Andorra
 - Austria
```
Comment (JohanFreelancer9) [inline Teams/rooms/microsoft-teams-rooms-premium.md:92]:
```suggestion
- Australia
- Austria
```

File: Teams/rooms/microsoft-teams-rooms-premium.md [hunk 2]
```diff
@@ -86,29 +86,86 @@ For more details on the Microsoft Teams Rooms managed service support coverage,
...
+- Cook Islands
+- Croatia
+- Cyprus
+- Czech Republic
 - Denmark
-- Finland
+- Estonia
+- Faroe Islands
+- Fiji
+- France
```
Comment (JohanFreelancer9) [inline Teams/rooms/microsoft-teams-rooms-premium.md:108]:
```suggestion
- Finland
- France
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by altsou** [comment]
All looks good to me! Thanks!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@dstrome PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**